config/defines.inc.php

```diff
@@ -32,11 +32,17 @@
 if (!defined('_PS_DISPLAY_COMPATIBILITY_WARNING_')) {
     define('_PS_DISPLAY_COMPATIBILITY_WARNING_', true);
 }
+if (!defined('_PS_DISPLAY_ONLY_ERRORS_')) {
+    define('_PS_DISPLAY_ONLY_ERRORS_', false);
+}
 if (_PS_MODE_DEV_ === true) {
     $errorReportingLevel = E_ALL | E_STRICT;
     if (_PS_DISPLAY_COMPATIBILITY_WARNING_ === false) {
         $errorReportingLevel = $errorReportingLevel & ~E_DEPRECATED & ~E_USER_DEPRECATED;
     }
+    if (_PS_DISPLAY_ONLY_ERRORS_ !== false) {
```

Comment:
You need to add this constant in two more places :
- In `phpstan.neon.dist` in `dynamicConstantNames` ,to remove the error PHPStan is complaining about (because it thinks the value is always false but it can be true, that why want it here)
- In `phpstan-disallowed-calls.neon` in `disallowedConstants` to block the calls of constant in the new, migrated code

Reply:
Thank you very much, I was stuck with that !